.github/actions/setup/action.yml

```diff
@@ -8,19 +8,14 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - uses: volta-cli/action@v4
-      env:
-        ACTIONS_STEP_DEBUG: truex 
-    - name: Get pnpm cache directory path
-      id: pnpm-cache-dir-path
-      run: echo "STORE_PATH=$(pnpm store path)" >> $GITHUB_OUTPUT
-      shell: bash
-    - uses: actions/cache@v3
-      id: pnpm-cache
+    - uses: pnpm/action-setup@v2
+      name: Install pnpm
+      with:
+        version: 7
```

Comment:
don't we want pnpm version 8?

Reply:
lol yes, but also I just noticed that our volta was pinning us to before the `resolution-mode setting is changed to highest` shenanigans so we'll see if anything breaks 🙃